modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js

```diff
@@ -629,6 +637,10 @@ AUI.add(
 			Menu,
 			'_registerMenu',
 			(event) => {
+				if (event.type === EVENT_KEYDOWN && event.which !== 32) {
```

Comment:
When debugging, I saw this event is a facade from AUI/YUI and it doesn't have `event.key`, so, this is why we're using deprecated `event.which`.

Does makes sense patching it on AUI? @markocikos 

-----

Hey @pat270, this is optional but you can use the constant

`A.Event.KeyMap.SPACE`

instead of 32

Reply:
I also noticed Chrome didn't send anything for `event.key` with space using this tool https://css-tricks.com/snippets/javascript/javascript-keycodes/#aa-tester-tool. This is why `event.which` was added.

`event.keyCode` is deprecated (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).

`event.which` is non-standard (https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/which)